HybrasylXML/XSD/Castables.Designer.cs

```diff
@@ -1367,8 +1367,6 @@ public Effects()
         {
             this._statuses = new Statuses();
             this._statModifiers = new StatModifiers();
-            this._damage = new Damage();
```

Comment:
Why were these removed?

Reply:
This conflicts with the way UseCastable was constructed. It evaluates Damage based on whether it's null or not. If it's null, it will then look at Heal and do the same thing. With these being set up here, Damage will never be null, so this evaluation fails. This is also related to the casting piece, so I will move these to a different PR.